tests/forcefields/test_md.py

```diff
@@ -143,8 +142,10 @@ def test_ml_ff_md_maker(
         for step in task_doc.objects["trajectory"].frame_properties
     )
 
-    with pytest.warns(FutureWarning):
-        name_to_maker[ff_name]()
+    # Skip the following test for DeepMD, since it doesn't have concrete implementations
```

Comment:
Is it possible to implement this? Or do you see any hurdles?

Reply:
This part of the code checks whether the concrete Maker classes raise a `FutureWarning`, indicating they will be deprecated. As per our earlier discussions, I’ve removed the concrete implementations for `DeepMD`, so this class no longer exists. Therefore, I’ve addressed this by skipping the test for `DeepMD`.